types/react-big-calendar/index.d.ts

```diff
@@ -6,7 +6,7 @@
 //                 Sebastian Silbermann <https://github.com/eps1lon>
 //                 Paul Potsides <https://github.com/strongpauly>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 2.9
```

Comment:
Why do we need to bump to 2.9?

Reply:
To use generic JSX elements as it is done in the test file.
This landed in TS 2.9 : https://github.com/Microsoft/TypeScript/pull/22415

Reply:
@eps1lon 

What is the implication of bumping to TypeScript 2.9  other than to get the CI build to run the test under this version?
Will that prevent consumers of this package from using it with TS 2.8 ?

I can always change the syntax in the test file to satisfy TS 2.8 if preferred.

Reply:
@eps1lon I have changed the syntax to satisfy TS 2.8. Are you happy with the PR ?

Reply:
You seem to have addressed his feedback.